### PR TITLE
Feature - Write read me and add nimble

### PR DESCRIPTION
**Pull request by at-lamle**

### CODE REVIEW

File: Nimble/NimbleDemoTests/QuickDemoTests.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//  NimbleDemoTests
+//
+//  Created by Lam Le V. on 1/15/18.
+//  Copyright © 2018 Lam Le V. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+final class QuickDemoTests: QuickSpec {
```
Comment (at-haiphung) [inline Nimble/NimbleDemoTests/QuickDemoTests.swift]: remove

File: Nimble/NimbleDemoTests/XCTestHelper.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+//  NimbleDemoTests
+//
+//  Created by Lam Le V. on 12/18/17.
+//  Copyright © 2017 Lam Le V. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+// overrides Swift global `fatalError`
```
Comment (at-haiphung) [inline Nimble/NimbleDemoTests/XCTestHelper.swift]: ///

File: Nimble/NimbleDemoTests/XCTestHelper.swift [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            expectation.fulfill()
+            unreachable()
+        }
+
+        // act, perform on separate thead because a call to fatalError pauses forever
+        DispatchQueue.global(qos: .userInitiated).async(execute: testcase)
+
+        waitForExpectations(timeout: 0.1) { _ in
+            // assert
+            XCTAssertEqual(assertionMessage, expectedMessage)
```
Comment (at-haiphung) [inline Nimble/NimbleDemoTests/XCTestHelper.swift]: use nimble

File: Nimble/NimbleDemo/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//
+//  Created by Lam Le V. on 12/18/17.
+//  Copyright © 2017 Lam Le V. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
```
Comment (at-haiphung) [inline Nimble/NimbleDemo/ViewController.swift]: remove unused code *

**Comment by at-tamnguyen3** [comment]
@at-lamle Hello bro, please take a look. I have just created a base content for `Nimble` document. I want you to follow these steps that I mention below:
- Close this pull request
- Create new `DemoNimble` project with Unit test only (do not include UI test)
- Define a `BasicDemoTests` class and implement some simple matchers with Nimble (do not implement with Quick)
- In README file, please implement section `So sánh giá trị`, `Kiểm tra kiểu` and `Kiểm tra tham chiếu`
- Then assign me in your new pull request
Thank you so much.

File: Nimble/README.md [hunk 5]
```diff
@@ -91,31 +97,249 @@ Bên cạnh đó còn có thể custom thông báo khi lỗi xảy ra:
...
+expect(actual).to(beGreaterThanOrEqualTo(expected))
+// Or
+expect(actual) >= expected
+```
 
-####Kiểm tra kiểu
+### Kiểm tra kiểu
 
-####Kiểm tra tham chiếu
+* `beAKindOf(aClass)`: Passes khi là có kiểu là `aClass`, passes khi là một thể hiện của `aClass` hoặc của bất kỳ lớp con nào từ `aClass`.
```
Comment (at-toantruong) [inline Nimble/README.md]: Passes khi có kiểu là `aClass`

**Pull request merged**